README.md

```diff
@@ -56,6 +56,7 @@ func main() {
 		ReferrerPolicy:        "strict-origin-when-cross-origin",
```

Comment:
help to remove the duplicate of line in README.

Reply:
Done, sorry about the delay.